packages/audits/src/components/AuditsTableIpCell.tsx

```diff
@@ -0,0 +1,143 @@
+    loadIpAddressMetadata();
+  }, []);
+
+  const loadIpAddressMetadata = async () => {
+    try {
+      setState({ ...state, loading: true });
+      const data = await api.metadata.getIpAdressMetadata(props.value);
+      setState({ data: data, loading: false });
+    } catch (e) {
+      console.log('failed to load metadata for ip address - ', e);
+    }
+  };
+
+  const { data, loading } = state;
+
+  const renderItems = (key: string) => {
+    const { data } = state;
+    switch (key) {
+      case 'city':
+        return (
+          <div key={key} className='fe-dflex fe-dflex-space-between'>
+            <span className={`${prefixCls}__ipCell-item`}>
+              <p className={`${prefixCls}__ipCell-item-name`}>City</p>
+              <p className={`${prefixCls}__ipCell-item-desc`}>{data[key] ?? 'unknown'}</p>
+            </span>
+            <span className={`${prefixCls}__ipCell-item`}>
+              <p className={`${prefixCls}__ipCell-item-name`}>Zip</p>
+              <p className={`${prefixCls}__ipCell-item-desc`}>{data.zip ?? 'unknown'}</p>
+            </span>
+          </div>
+        );
+      case 'latitude':
+        return (
+          <span key={key} className={`${prefixCls}__ipCell-item`}>
+            <p className={`${prefixCls}__ipCell-item-name`}>Latitude, Longitude</p>
+            <p className={`${prefixCls}__ipCell-item-desc fe-mb-0`}>
+              {data.latitude ? `${data.latitude.toFixed(4)},` : 'unknown'}
+              {data.longitude ? data.longitude.toFixed(4) : ''}
+            </p>
+          </span>
+        );
+      case 'country_name':
+        return (
+          <span key={key} className={`${prefixCls}__ipCell-item`}>
+            <p className={`${prefixCls}__ipCell-item-name`}>Country</p>
+            <p className={`${prefixCls}__ipCell-item-desc`}>{data[key] ?? 'unknown'}</p>
+          </span>
+        );
+      default:
+        return;
+    }
+  };
+
+  return (
+    <Popup
+      action='click'
+      trigger={
+        <div className={`${prefixCls}__ipCell`}>
+          <div className={`${prefixCls}__ipCell-countryFlag`}>
+            {loading ? <Loader size={15} /> : data.location.country_flag_emoji ?? ''}
+          </div>
+          <div className={`${prefixCls}__ipCell-ipAddress`}>{props.value}</div>
+        </div>
+      }
+      content={
+        <div className={`${prefixCls}__ipCell-window`}>
+          <div className={`${prefixCls}__ipCell-window-title`}>IP ADDRESS {props.value}</div>
+          <div className={`${prefixCls}__ipCell-window-container fe-dflex-space-between`}>
+            <div className={`${prefixCls}__ipCell-window-info`}>
+              {Object.keys(data).map((key) => {
+                return renderItems(key);
+              })}
+            </div>
+            <div className={`${prefixCls}__ipCell-window-map`}>
+              {data && (
+                <GoogleMapReact
+                  bootstrapURLKeys={{ key: 'AIzaSyC588_3ox-RV9tt46VXdRkaPpSSZbSDAfE' }}
```

Comment:
move the bootstrapUrlKeys to ContextHolder with default value, so vendor can set there keys